alembic/versions/0595e877f471_add_streamphotometry_table.py

```diff
@@ -0,0 +1,63 @@
+"""Add StreamPhotometry table
```

Comment:
Why is this committed here?

Reply:
I think what happened is I tried to re-do the migration after pulling in master into my master, but I didn't pull master into this new branch. 

Next time I will stamp my database before I start working...